src/web-components/who-is-online/components/dropdown.test.ts

```diff
@@ -0,0 +1,273 @@
+import '.';
+import { MeetingColorsHex } from '../../../common/types/meeting-colors.types';
+import sleep from '../../../common/utils/sleep';
+import { Participant } from '../../../components/who-is-online/types';
+
+interface elementProps {
+  position: string;
+  align: string;
+  participants?: Participant[];
+  label?: string;
+  returnTo?: string;
+  options?: any;
+  name?: string;
+  icons?: string[];
+}
+
+const mockParticipants: Participant[] = [
+  {
+    avatar: {
+      imageUrl: '',
+      model3DUrl: '',
+    },
+    color: MeetingColorsHex[0],
+    id: '1',
+    name: 'John Zero',
+    slotIndex: 0,
+  },
+];
+
+const createEl = ({
+  position,
+  align,
+  label,
+  returnTo,
+  options,
+  name,
+  icons,
+  participants,
+}: elementProps): HTMLElement => {
+  const element: HTMLElement = document.createElement('superviz-who-is-online-dropdown');
+
+  if (label) {
+    element.setAttribute('label', label);
+  }
+
+  if (returnTo) {
+    element.setAttribute('returnTo', returnTo);
+  }
+
+  if (name) {
+    element.setAttribute('name', name);
+  }
+
+  if (icons) {
+    element.setAttribute('icons', JSON.stringify(icons));
+  }
+
+  if (participants?.length) {
+    element.setAttribute('participants', JSON.stringify(participants));
+  }
+
+  if (options) {
+    element.setAttribute('options', JSON.stringify(options));
+  }
+
+  if (!options) {
+    const defaultOptions = [
+      {
+        name: 'EDIT',
+        value: {
+          uuid: 'any_uuid',
+        },
+      },
+      {
+        name: 'DELETE',
+        value: {
+          uuid: 'any_uuid',
+        },
+      },
+    ];
```

Comment:
What's this? 


```suggestion
  if (!options) {
    const defaultOptions = [
      {
        name: 'EDIT',
        value: {
          uuid: 'any_uuid',
        },
      },
      {
        name: 'DELETE',
        value: {
          uuid: 'any_uuid',
        },
      },
    ];
```

Reply:
Just something that was useful in tests of superviz-dropdown, that I copied and failed to verify that is still necessary